server/models/Column.js

```diff
@@ -0,0 +1,24 @@
+const mongoose = require("mongoose");
+
+const columnSchema = new mongoose.Schema(
+  {
+    name: {
+      type: Array,
+      required: false,
+    },
+    cards: {
+      type: Array,
+      required: true,
+    },
+  },
+  { timestamps: true }
+);
+
+columnSchema.methods.removePassword = function () {
+  const object = this.toObject();
+  delete object.user.password;
+  return object;
+};
+
+const Column = mongoose.model("column", columnSchema);
+module.exports = Column;
```

Comment:
In our other model files we are exporting a bit differently, this is not the same as our other files. `module.exports = { Column: mongoose.model("column", columnSchema), columnSchema }` is more consistent. In the future, we should refactor to bundle all models together and make them importable from an index file.

Reply:
I'll work on this with @street-jackal!

Reply:
updated across the *board*